### ISSUE TITLE
First property of inline style missing trailing comma

### ISSUE THREAD

**Issue by jesstelford** [roman01la/html-to-react-components#9]
On commit 775e9b12064e0dcae0f1eef9d5fa21c0ff461700

This minimal test case:

```javascript
console.log(require('html-to-react-components')(
  '<div data-component="Component" style="padding:0; display:inline"></div>'
).Component);
```

Produces this output:

```javascript
import React from 'react';

const Component = React.createClass({
  render() {
    return <div style={{
        padding: 0
        display: 'inline'
      }} />;
  }
});
```

Notice the `padding: 0` line is missing a trailing comma.

When more than two inline style elements exist, commas are correctly inserted for properties other than the first. For example:

```javascript
console.log(require('html-to-react-components')(
  '<div data-component="Component" style="padding:0; display:inline; margin: 10; border: none"></div>'
).Component);
```

Produces

```
import React from 'react';

const Component = React.createClass({
  render() {
    return <div style={{
        padding: 0
        display: 'inline',
        margin: 10,
        border: 'none'
      }} />;
  }
});
```

**Comment by jesstelford** [comment]
Note that I tried the same case in `htmltojsx`, and it gives the correct output:

```javascript
var HTMLtoJSX = require('htmltojsx');
var converter = new HTMLtoJSX({
  createClass: false
});
console.log(converter.convert('<div data-component="Component" style="padding:0; display:inline; margin: 10; border: none"></div>'));
```

Produces

```
<div data-component="Component" style={{padding: 0, display: 'inline', margin: 10, border: 'none'}} />
```

**Comment by roman01la** [comment]
Interesting. I'm going to check this. Thanks!

**Comment by roman01la** [comment]
fixed in 1ecc6998

**Comment by jesstelford** [comment]
:+1: Thanks!
